docs/en/05_Releases_and_changelogs/cwp_recipe_basic_1.0.4.md

```diff
@@ -5,17 +5,17 @@
 This release tracks the Framework release 3.1.6 and fixes some module bugs. The highlights are:
 
 * Addition of five new modules:
-  * [external links checker](https://github.com/silverstripe-labs/silverstripe-externallinks)
+  * [external links checker](https://github.com/silverstripe/silverstripe-externallinks)
```

Comment:
Don't modify the old changelogs, they can stay as they were at time of writing =)